next/components/paragraph--image.tsx

```diff
@@ -1,6 +1,6 @@
-import { MediaImage } from "@/components/media--image";
+import { Media } from "@/components/media";
 import { Image } from "@/lib/zod/paragraph";
 
 export function ParagraphImage({ paragraph }: { paragraph: Image }) {
-  return <MediaImage media={paragraph.field_image} priority />;
+  return <Media media={paragraph.field_image} priority={true} />;
```

Comment:
Since we have the generic <Media> component, I think we can use that instead of the more specific one in these components? 

(I have changed all of them)

Reply:
Seems good to me at this stage.